eslint/comments.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   extends: [
-    'plugin:eslint-comments/recommended',
```

Comment:
I replaced `eslint-comment` to `@eslint-community/eslint-comments` cause eslint-comment is not maintained for a while